Source/Core/DolphinQt2/Config/Mapping/MappingButton.cpp

```diff
@@ -130,7 +132,7 @@ void MappingButton::OnButtonTimeout()
 void MappingButton::Clear()
 {
   m_reference->SetExpression("");
-  m_reference->range = 100;
+  m_reference->range = 100. / SLIDER_TICK_COUNT;
```

Comment:
Should probably make this `100.0` so it's not as easy to unintentionally gloss over when reading